gyp/gyp_main.py

```diff
@@ -17,7 +17,7 @@ def IsCygwin():
     stdout,stderr = out.communicate()
     return "CYGWIN" in stdout
   except Exception, e:
```

Comment:
Old style exceptions are syntax errors in Python 3 while new style exceptions work as expected in both Python 2 and Python 3 so please consider changing the line above to __except Exception as e:__

Reply:
This has been addressed